Documentation/supermuc.rst

```diff
@@ -36,100 +29,36 @@ Load module. Add these lines to .bashrc:
   export CXX=mpiCC 
   export FC=mpif90
 
- 
-Alternatively (and for reference), to compile seissol-env on SuperMUC-NG, follow the procedure below:
-
-.. code-block:: bash
-
-    # load spack
-    module load user_spack
-    # clone seissol-spack-aid and add the repository
-    # we use a supermuc specific branch as supermuc spack is too old (0.17.1) for the main branch
-    git clone --branch supermuc_NG https://github.com/SeisSol/seissol-spack-aid.git
-    cd seissol-spack-aid
-    spack repo add ./spack
-    # locate externally build pkg-config
-    spack external find pkg-config
-
-    # install all dependencies of seissol.
-    # We specify the intel and intel-mpi version matching preinstalled version on supermuc-ng
-    # These can be found with:
-    # >spack find intel-mpi
-    # >spack compiler list
-    spack install seissol-env %intel@21.4.0 ^intel-mpi@2019.12.320
-    # or
-    spack install seissol-env %gcc@11.2.0 ^intel-mpi@2019.12.320
-
-    # now create a module:
-    spack module tcl refresh seissol-env@develop%intel@21.4.0
-
-    #to access the module at start up, add to your ~/.bashrc
-    module use $HOME/spack/modules/x86_avx512/linux-sles15-skylake_avx512/
-    # change this path to your_custom_path_2_modules if you update ~/.spack/modules.yaml 
-
-Custom install directory for packages and modules can be set with, by changing ``~/.spack/config.yaml``:
-
-.. code-block:: yaml
-
-    config:
-      install_tree: path_2_packages
-
-and ``~/.spack/modules.yaml``: 
-
-.. code-block:: yaml
-
-    modules:
-      default:
-        roots:
-         tcl: your_custom_path_2_modules
 
-This can be useful to share packages with other user of a SuperMUC project.
+Install pspamm (see :ref:`pypi_behind_firewall` for the proxy):
```

Comment:
Is it maybe useful to also insert a comment about `--no-build-isolation` into that article?

Reply:
(we could also recommend/suggest using a virtual environment for the Python packages, if it helps—but maybe that's a bit out of scope for this topic here)